.github/render-charts.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eo pipefail
+
+chartYaml="${1}"
+chartName=$(dirname "${chartYaml}")
+
+echo "::group::Render chart ${chartName}"
```

Comment:
Any reason for these 2 `echo` lines?

Reply:
yes, this is a special echo recognized by GitHub to group output to stdout. See example here: https://github.com/vshn/appcat-service-postgresql/runs/6426139691?check_suite_focus=true, in the step "Render Helm charts" there's a clickable "Render chart charts/provider-postgresql" that toggles the output of the `helm template` per chart :)
This neatly organizes the output of multiple charts, if there are any in the future (e.g. forks or so), otherwise the output will be very hard to read ("to which chart belongs this file again...?").

Reply:
Great feature, thanks